.github/workflows/ci.yml

```
@@ -630,3 +630,111 @@ jobs:
           notify_when: 'failure'
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_NOTIFY_BUILD }}
+
+  build-and-publish-forc-binary:
+    name: Forc Binary - ${{ matrix.job.target }} (${{ matrix.job.os }})
```

The matrix doesn't actually exist at this line, so it looks like `Forc Binary - ${{ matrix.job.target }} (${{ matrix.job.os }})` (literally).
```suggestion
    name: Build and upload Forc binaries
```